rest.go

```diff
@@ -75,6 +78,33 @@ func getBaseUrl() string {
 	return DEFAULT_BASE_URL
 }
 
+func structToURLValues(input interface{}) string {
+	queryValues := url.Values{}
+	structType := reflect.TypeOf(input)
+	structValue := reflect.ValueOf(input)
+
+	for i := 0; i < structType.NumField(); i++ {
+		field := structType.Field(i)
+		value := structValue.Field(i)
+
+		if !value.IsNil() {
+			queryParam := field.Tag.Get("queryParam")
```

Comment:
Can we make it simpler than `"queryParam"`?
Maybe just `query` or `param` is enough